services/wallet/thirdparty/alchemy/client.go

```diff
@@ -0,0 +1,152 @@
+package alchemy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/status-im/status-go/services/wallet/bigint"
+	"github.com/status-im/status-go/services/wallet/thirdparty"
+)
+
+func getBaseURL(chainID uint64, apiKey string) (string, error) {
+	switch chainID {
+	case 1, 5, 11155111:
```

Comment:
I think we need some global const ids for chains.

Reply:
Super-agree. What's the right place for that in status-go? Never ventured too far from the wallet service

Reply:
Done